class NotFound(Exception):
    pass


class InternalServerError(Exception):
    pass
